lib/base/RequestClient.js

```diff
@@ -133,7 +133,10 @@ function logRequest(options) {
 
   if (options.headers) {
     console.log('Headers:');
-    console.log(options.headers)
+    const headers = Object.keys(options.headers).filter((header) => {
```

Comment:
I think we might want to add a test here.

Reply:
Should it be a unit test in the Request.Client.spec? and should the test be comparing the logging output or just checking that `headers` does not contain any auth headers?

Reply:
I like a unit test in `Request.Client.spec`, minimally checking that `headers` does not contain any forbidden data. Thanks!